src/lib/bits/alert-dialog/components/AlertDialogAction.svelte

```diff
@@ -24,6 +24,7 @@
 		use:melt={builder}
 		{...$$restProps}
 		{...attrs}
+		type="button"
```

Comment:
For the rest of the buttons, I think we should do something like this:

```suggestion
		type="button"
		{...$$restProps}
		{...attrs}
```

That way, if for some reason or another someone wants to submit the form when they close the dialog, or toggle a switch, or whatever, they are able to override the `type="button"` we set by passing `type="submit"` since the `...$$restProps` follows our hardcoded type.

Reply:
done as well.

I also checked other buttons with same mindset in mind and one place caught my, and I leave it just to consider: 

You have DialogTrigger, DialogClose, but maybe there should be as well DialogAction, similar to AlertDialogAction?

I also saw you're working on Combobox for bits, and DatePicker for melt-ui, please remember of type="button" there :)

Reply:
Me again. I just read again your comments and you suggested to allow pass type to all of them? I just added to TriggerAction and AlertDialogAction. Do you want me to add same to other as well? I reviewed my code and I'm not sure if other buttons needs it :) I think not... Maybe CollapsibleTrigger, but tbh I didn't find good-enough reason.

Reply:
Yeah it won't hurt to add it to all except the select tbh, it doesn't hurt by giving them the option to override!

Reply:
Ok, I will add them, but still not sure about radio and checkboxes. Form inputs should not trigger submit imho.

Reply:
Ok, done. I also find a reason to set type="submit" on radio/checkboxes as well. Like simple form with just radio, and if user clicks on option it will also trigger form submit.

But I thought of something else. If SelectTrigger has type=button which cannot be changed, maybe popover and menu bar should not as well?
Or maybe even for Select we should leave a way to change SelectTrigger type.

Reply:
Yeah I'm cool with leaving a way to override it's type like the others tbh! More flexibility the better here! @wysher

Reply:
Thanks and done. All buttons have type="button" by default which can be changed by dev (including SelectTrigger)